### DISCUSSION TITLE
Didn't recieve the galaxy brain badge !

### DISCUSSION THREAD

**Discussion by HOTHEAD01TH** [community/community#146131] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

hey guys i have 2 accepted answers in github community why didnt i recieved the galaxy brain badge ?
![Screenshot 2024-12-03 184821](https://github.com/user-attachments/assets/0e2f6c81-f08d-4de9-ae4c-e433f6825f3e)

**Comment by davevad93** [comment, marked as answer]
Hi @HOTHEAD01TH ,

Unfortunately, achievements are no longer available in GitHub Community Discussions, but you can still earn the badge by replying to discussions outside the community, for example in repos with open discussions. For more information on why achievements have been disabled, see the following post:

- [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536)

**Reply by HOTHEAD01TH** [reply]
Thank you for sharing this update and providing a transparent explanation of why Achievements in GitHub Community Discussions have been disabled. It’s unfortunate to see how misuse of the system has led to this decision, but it’s understandable that maintaining the quality and integrity of discussions is a priority.
